samples/TCPClient6.js

```diff
@@ -31,12 +31,12 @@ net.on('up', function() {
 		function connected() {
 			console.log("connect successful");
 			is_connected = true;
-			client.write(new Buffer("initial data"));
+			client.write(new Buffer("initial data\r\n"));
 		}
 
 		client.on("data", function(data) {
 			console.log("got data: " + data.toString('ascii'));
-			client.write(new Buffer("write data " + count++));
+			client.write(new Buffer("write data " + count++ + "\r\n"));
```

Comment:
maybe do the counter inrement on a different line, then append to the string, it's probably cleaner than ++ +